### PR TITLE
new create campaign design

### PR DESCRIPTION
**Pull request by dpolozov**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -27,4 +27,4 @@ yarn-error.log*
 
 .idea/
 .eslintcache
-.vscode/
+.vscode/launch.json
```
Comment (grishick) [inline .gitignore]: I meant to exclude the whole .vscode/ folder, not just launch.json

File: src/components/CreateCampaign2.js [hunk 2]
```diff
@@ -1,44 +1,49 @@
...
-import {Container, Form, Col, Button, Image} from 'react-bootstrap';
+import {Container, Form, Col, Button, Image, Modal} from 'react-bootstrap';
 import ReactPlayer from 'react-player';
 import config from "react-global-configuration";
 import uuid from 'react-uuid';
 import axios from 'axios';
-import { ChevronLeft } from 'react-bootstrap-icons';
-var HEOCampaignFactory, ACCOUNTS, web3;
+import { ChevronLeft, CheckCircle, ExclamationTriangle, HourglassSplit, XCircle } from 'react-bootstrap-icons';
+var HEOCampaignFactory, HEOGlobalParameters, HEOPriceOracle, ACCOUNTS, web3;
```
Comment (grishick) [inline src/components/CreateCampaign2.js]: no need to add back "HEOGlobalParameters, HEOPriceOracle,"

File: server/package.json [hunk 3]
```diff
@@ -15,6 +15,7 @@
     "express": "^4.17.1",
     "express-fileupload": "^1.2.1",
     "node-fetch": "^2.6.1",
+    "throng": "^5.0.0",
```
Comment (grishick) [inline server/package.json:18]: this is needed by worker.js

File: src/components/CreateCampaign2.js [hunk 4]
```diff
@@ -1,44 +1,49 @@
...
+            maxAmount:10000,
+            title:"",
             description:"Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor.",
             raisedAmount:0,
             tokensToBurn:0,
             percentRaised: "0%",
             mainImageURL: "",
             metaDataURL:"",
             mainImageFile:"",
+            reward:0,
```
Comment (grishick) [inline src/components/CreateCampaign2.js]: this is not used

File: src/components/CreateCampaign2.js [hunk 5]
```diff
@@ -273,9 +324,10 @@ class CreateCampaign2 extends React.Component {
             alert("Please install metamask");
         }
         
-
+    
         let options = (config.get("chainconfigs")[config.get("CHAIN")]["currencyOptions"]);
         this.setState({coinOptions : options, currencyAddress:options[0].value, currencyName:options[0].text});
+
```
Comment (grishick) [inline src/components/CreateCampaign2.js:328]: why an extra empty line here?

File: src/css/createCampaign.css [hunk 6]
```diff
@@ -108,4 +108,41 @@
 #createCampaignVideoPlayer{
     width: 20rem !important;
     height: auto !important;
+
```
Comment (grishick) [inline src/css/createCampaign.css]: nit: extra line

**Review by grishick — CHANGES_REQUESTED** [review]
looks like some changes got snuck back in via the rebase

File: server/package.json (hunk 3 above)
Comment (dpolozov) [reply server/package.json:18]: that one shows in mine as being there

**Review by grishick — APPROVED** [review]

**Pull request merged**